chainerx_cc/chainerx/python/pymodule.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <pybind11/pybind11.h>
+
+#include "chainerx/macro.h"
+
+namespace chainerx {
+namespace python {
+namespace python_internal {
+
+namespace py = pybind11;
+
+inline py::handle& numpy_module() {
```

Comment:
We use camel case for free functions.
How about `GetCachedNumpyModule`, and so on?